components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -0,0 +1,138 @@
++++
+title = "Data Feeds"
+description = "Export Node Data from Chef Client Runs to External Data Collectors"
+date = 2020-05-05T13:19:02-07:00
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "settings"
+    weight = 15
++++
+
+The Data Feed service sends node data to a 3rd party service.
+This can be useful when updating Configuration Management Databases, external Security Dashboards and IT service management platforms.
+The following types of information are sent:
+
+- Ohai data gathered from each managed node - This data includes hardware, operating system, and installed program information. Some variation depends on the managed operating system
+- Configuration information about each managed node - This information includes Chef Client Run status, Runlists, Cookbooks, and Recipes being ran against each node
+- Compliance information about each node that shows the compliance state - This information includes passed and failed controls for each profile executed against that node
+
+A Data Feed is not a data tap.
+A Data Feed operates by doing the following:
+
+- Every 4 hours, the data-feed-service will aggregate the client runs and compliance reports from the previous 4 hours and send this information to the registered destinations. This time interval is 4 hours by default, but is configurable
+- If there are no destinations, aggregation will not occur
+- The data aggregates and sends in batches of 50 nodes at a time. The batch amount is 50 by default, but is configurable
+
+By default, only Admin users of Chef Automate may create and manage Data Feeds.
+
+### Adding a Data Feed Instance
+
+A single Data Feed instance connects to one 3rd party endpoint.
+Create as many Data Feed instances as needed.
+
+To add a Data Feed instance in Chef Automate:
+
+![Setup Data Feed Page](/images/docs/filled_form_create_data_feed.png)
+
+1. In the **Settings** tab, navigate to _Data Feeds_ in the sidebar
+2. Select **Create Data Feed**
+3. Enter a unique Data Feed name
+4. Enter the URL for your Data Feed End Point, including any specific port details
+5. Enter the Username and Password that your 3rd party endpoint requires for authentication
+6. Select **Test Data Feed** to begin validating the connection details
+7. Once the test is successful, select **Create Data Feed** to save your Data Feed configuration
+
+### Edit a Data Feed Instance
+
+To edit a Data Feed instance in Chef Automate:
+
+1. In _Data Feeds_, select the Data Feed name to open its detail page
+1. Edit the Data Feed name or URL
+1. Use the **Save** button to save your changes
```

Comment:
Should choose a single style of numbered lists for consistency. Here you use the 1, 1, 1 style, but above (L40-46) you use sequential numbers. There is no difference in the rendered output, so not a big deal, just a consistency thing, as I said. (My preference is to use sequential numbers because then readers of the raw file get to see the sequence, too.)

Reply:
Thank you for catching that, @msorens. The style guide designates using the 1,1,1,1 style, but the text editor sometimes fills in the numbers in a list automatically. Fixing this.